src/store/actions.js

```diff
@@ -499,6 +500,10 @@ export default {
 					// Already removed from unified inbox
 				})
 
+				dispatch('fetchUnreadStats', {
```

Comment:
so this means one additional request per mailbox sync?

What if we enhance the *envelope sync* into a *mailbox sync* that returns the latest stats of the mailbox *and* the new/changed/vanished messages?

Reply:
Good idea. That extra request is obsolete.